### ISSUE TITLE
False-positive STRING_TEMPLATE_CURLY_BRACES

### ISSUE THREAD

**Issue by petertrr** [cqfn/diKTat#401]
```kotlin
        val copyTestFile = File("${testFile}_copy")
```
gets corected to
```kotlin
        val copyTestFile = File("$testFile_copy")
```

**Comment by petertrr** [comment]
Other examples:
`fun warnText() = "[${ruleName()}] ${this.warn}:"`
`.run { if (size > 1) "${first()}..." else first() }`
